code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -629,8 +629,8 @@ Basically, we fill the time between now and 2s from now with hands based off the
 	if(!(DT_PROB(creation_purity*10, delta_time)))
 		return
 	var/traumalist = subtypesof(/datum/brain_trauma)
-	traumalist -= /datum/brain_trauma/severe/split_personality //Uses a ghost, I don't want to use a ghost for a temp thing.
-	traumalist -= /datum/brain_trauma/special/obsessed //Sets the owner as an antag - I presume this will lead to problems, so we'll remove it
+	var/list/forbiddentraumas = list(/datum/brain_trauma/severe/split_personality, /datum/brain_trauma/special/obsessed, /datum/brain_trauma/hypnosis) // Split personality uses a ghost, I don't want to use a ghost for a temp thing; Obsessed sets the owner as an antag - I presume this will lead to problems, so we'll remove it; Hypnosis, same reason as obsessed, plus a bug makes it remain even after the neurowhine purges and then turn into "nothing" on the med reading upon a second application
+	traumalist -= forbiddentraumas
```

Comment:
I think I prefer removal and reasoning on the same line, split up like before. Was the target cleanliness?

Reply:
It was, but you're right. Ordering it like this is a bit unwieldy, I'll split it up